server-backend/src/SwapOutController.ts

```diff
@@ -44,6 +46,16 @@ export class SwapOutController {
         return this.mapToResponse(swap);
     }
 
+    @Post('/liquid')
```

Comment:
I don't think we need a separate endpoint per chain, IMO, we should only have `/swap/out` and in the request body they should indicate to what chain.

Reply:
I suggest to keep it separated in order to avoid a lot of if/else in the code 🍝 but of course we can do it that way. Let me know what you prefer

Reply:
I think the primary goal of an API is to make it consistent and easily usable by clients.
If we merge the 2 endpoints into one, we'll just need 1 extra if-else.
```suggestion
    @Post('/out')
    swapOut(request: SwapOutRequest): Promise<SwapOutResponse> {
        if (request.chain === 'BITCOIN) {
            createBitcoinSwapOut();
        } else if (request.chain === 'LIQUID') {
            createLiquidSwapOut();
        }
     }
```

Reply:
Done